src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -1,18 +1,60 @@
 package wolox.training.controllers;
 
-import org.springframework.stereotype.Controller;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 import org.springframework.ui.Model;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.*;
+import wolox.training.exceptions.BookIdMismatchException;
+import wolox.training.exceptions.BookNotFoundException;
+import wolox.training.models.Book;
+import wolox.training.repositories.BookRepository;
 
 
-@Controller
+@RestController
+@RequestMapping("/api/books")
```

Comment:
What about versioning? Do we do that here?

Reply:
Up until now, I don't recall a project where we had to do API versioning... so don't worry about it for now and I'll ask the team if there's any case that we needed to do it and I'll tell you.